### ISSUE TITLE
Unable to use Set in Deno module

### ISSUE THREAD

**Issue by affanshahid** [exograph/exograph#1183]
I'm trying to use `Set` to refer to lists in a Deno module but I get the following error:
```
error[C000]: Type Set was used as an output type somewhere else in the model. Types may only be used as either an input type or an output type.
  --> src/index.exo:32:58
   |
32 |   mutation chat(@inject config: ConfigContext, messages: Set<Message>): String
   |                                                          ^^^ conflicting usage
...
44 |   mutation generateQuestions(projectId: Uuid): Set<Uuid>
   |                                                --- an output type
```

The workaround is to use some indirection:

```
  type Input {
    messages: Set<Message>
  }

  type Message {
    role: String
    content: String
  }

mutation chat(@inject config: ConfigContext, messages: Input): String

mutation generateQuestions(projectId: Uuid): Set<Uuid> // <- the alleged duplicate
```

**_Edit_: It seems even the workaround does not work.** Using `Set` directly in inputs causes weird behaviors.

I can do:
```
mutation chat(@inject config: ConfigContext, messages: Set<String>): String
```
But not:
```
mutation chat(@inject config: ConfigContext, messages: Set<Input>): String
```
If I do this, the Playground refuses to connect to the API. So not sure what the issue is.

**Comment by ramnivas** [comment]
@affanshahid Thanks for reporting. I have fixed this and released 0.9.4 with it. Please try it out.

**Comment by affanshahid** [comment]
Thanks works perfectly now 👍🏻
